fix: LEAP-1692: Image export for COCO and YOLO

robot-ci-heartex opened the pull request:
Hi @triklozoid!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/383 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12632729436)

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12632729436

triklozoid committed:
Api changes

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12670993624

triklozoid committed:
Pass parameters to converter

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12673643838

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12675815238

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12677921549

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12691517572

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12746063666

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12746762647)
> Successfully merged:  72 files changed, 297 insertions(+), 742 deletions(-)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12746762647

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12747261339

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12750826465

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12750829884

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12750897565

MihajloHoma commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12768462723)
> Successfully merged:  59 files changed, 586 insertions(+), 699 deletions(-)

MihajloHoma committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12768462723

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12790705182

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12791627454)
> Successfully merged:  create mode 100644 label_studio/core/static/templates/medical-image-classification.png

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12791627454

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12796261716

MihajloHoma commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12804635117)
> Successfully merged:  create mode 100644 web/libs/core/src/lib/Tour/index.ts

MihajloHoma committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12804635117

triklozoid committed:
Fix test and cloud export

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12808238989

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12832216866

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12832272422)
> Successfully merged:  13 files changed, 214 insertions(+), 105 deletions(-)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12832272422

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12885355983)
> Successfully merged:  create mode 100644 web/libs/editor/src/utils/billing.ts

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1692'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12885355983

triklozoid committed:
Fix poetry.lock

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/12886052809
